Makefile

```diff
@@ -58,12 +58,14 @@ test-compile-changed: os-autoinst/
 
 .PHONY: test-yaml-valid
 test-yaml-valid:
-	@# Get list of changed yaml files
-	$(eval YAMLS=$(shell sh -c "git --no-pager diff --diff-filter=dr --name-only master | grep '\(schedule\|test_data\)/.*\.y.\?ml$$'"))
-	if test -n "$(YAMLS)"; then \
-	  export PERL5LIB=${PERL5LIB_} ; tools/test_yaml_valid $(YAMLS);\
+	@# Get list of changed yaml files. We only want to lint changed files for
```

Comment:
`@#` :raised_eyebrow: Not something you introduced I guess but it seems odd to me to hide comments specifically here.